changelogs/fragments/3374-add-ipa-ptr-sync-support.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - ipa - dns zone synchronization support (https://github.com/ansible-collections/community.general/pull/3374).
```

The module name is ipa_dnszone:
```suggestion
  - ipa_dnszone - add DNS zone synchronization support (https://github.com/ansible-collections/community.general/pull/3374).
```